### ISSUE TITLE
Save trimmed parameter saying 'false' is not a valid choice

### ISSUE THREAD

**Issue by rvijay-seqcoast** [nf-core/bacass#128]
### Description of the bug

I am running into an issue with the --save_trimmed_fail setting when running the bacass pipeline. It indicates that 'false' is not a valid choice, but it also indicates that 'true' or 'false' are the only valid arguments for this setting.

### Command used and terminal output

```console
nextflow run https://github.com/nf-core/bacass \
		 -name bacass-test-2 \
		 -params-file https://api.cloud.seqera.io/ephemeral/Mn6cNSXHpkJNUK5IW3P4HQ.json \
		 -with-tower \
		 -r 2.2.0 \
		 -profile docker


  workDir        : /seqcoast-aws/scratch/1kcilKqDAJaXkO
  projectDir     : /.nextflow/assets/nf-core/bacass
  userName       : root
  profile        : docker
  configFiles    :
Input/output options
  input          : https://raw.githubusercontent.com/nf-core/test-datasets/bacass/bacass_hybrid.tsv
  outdir         : s3://seqcoast-aws
Assembly parameters
  assembly_type  : hybrid
  canu_mode      : -nanopore
Annotation
  dfast_config   : /.nextflow/assets/nf-core/bacass/assets/test_config_dfast.py
Skipping Options
  skip_kraken2   : true
!! Only displaying parameters that differ from the pipeline defaults !!
------------------------------------------------------
If you use nf-core/bacass for your analysis please cite:
* The pipeline
  10.5281/zenodo.2669428
* The nf-core framework
  https://doi.org/10.1038/s41587-020-0439-x
* Software dependencies
  https://github.com/nf-core/bacass/blob/master/CITATIONS.md
------------------------------------------------------
ERROR ~ ERROR: Validation of pipeline parameters failed!
-- Check 'nf-1kcilKqDAJaXkO.log' file for details
The following invalid input values have been detected:
* --save_trimmed_fail: 'false' is not a valid choice (Available choices: true, false)
-- Check script '.nextflow/assets/nf-core/bacass/./workflows/../subworkflows/local/utils_nfcore_bacass_pipeline/../../nf-core/utils_nfvalidation_plugin/main.nf' at line: 57 or see 'nf-1kcilKqDAJaXkO.log' file for more details
```

### System information

- Nextflow version: 23.10.1
- Hardware: AWS
- Executor: awsbatch
- Container engine: Docker
- OS: Linux
- Version of nf-core/bacass: 2.2.0

**Comment by Daniel-VM** [comment]
Hi @rvijay-seqcoast!, this issue is already fixed in dev (see #125 ). Incoming release will have this fix available in a tagged version (future 2.3.0). 

In the mean time you can try `-r dev`: 
`
nextflow run https://github.com/nf-core/bacass  \
    -name bacass-test-2 \
    -params-file https://api.cloud.seqera.io/ephemeral/Mn6cNSXHpkJNUK5IW3P4HQ.json \
    -with-tower \
    -r dev \
    -profile docker
`

Hope this help. Let me know if that works 😃

**Comment by Daniel-VM** [comment]
clossing this. re-open it if the issue persist
